pages/dao/[symbol]/proposal/components/instructions/Vsr/VotingMintConfig.tsx

```diff
@@ -52,76 +58,147 @@ const VotingMintConfig = ({
   const { realm } = useRealm()
   const { assetAccounts } = useGovernanceAssets()
   const shouldBeGoverned = !!(index !== 0 && governance)
-  const [form, setForm] = useState<ConfigureCollectionForm>()
   const [formErrors, setFormErrors] = useState({})
+  const [showHeliumFields, setShowHeliumFields] = useState(false)
+  const [form, setForm] = useState<ConfigureVotingMintForm>()
   const { handleSetInstructions } = useContext(NewProposalContext)
   const { wallet, anchorProvider } = useWallet()
 
+  useEffect(() => {
+    if (form?.programId) {
+      if (vsrPluginsPks.includes(form.programId)) {
+        setShowHeliumFields(false)
+      }
+
+      if (heliumVsrPluginsPks.includes(form.programId)) {
+        setShowHeliumFields(true)
+      }
+    }
+  }, [form?.programId, setShowHeliumFields])
```

Comment:
```suggestion
  const showHeliumFields = useMemo(() => heliumVsrPluginsPks.includes(form.programId) 
  }, [form?.programId, vsrPluginPks heliumVsrPluginsPks])
```
Would this work? Seems like this should be a memoized computation, and not be in state.